include/quic/address.hpp

```diff
@@ -5,6 +5,8 @@
 
 namespace oxen::quic
 {
+    constexpr std::array<uint8_t, 16> _ipv6_any_addr = {0};
```

Comment:
This should be `inline constexpr`

Reply:
(Or alternatively, moved into the .cpp along with the `is_any_addr` body)

Reply:
fixed thank you